test_schemas/en/test_interviewer_note.json

```diff
@@ -0,0 +1,121 @@
+                                "interviewer_note": true,
+                                "contents": [
+                                    {
+                                        "description": "In this section we will ask you about your favourite team."
+                                    }
+                                ]
+                            },
+                            "type": "Interstitial"
+                        },
+                        {
+                            "type": "Question",
+                            "id": "favourite-team-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "favourite-team-answer",
+                                        "label": "Favourite team",
+                                        "mandatory": false,
+                                        "q_code": "0",
+                                        "type": "TextField"
+                                    }
+                                ],
+                                "id": "favourite-team-question",
+                                "title": "Who is your favourite team?",
+                                "interviewer_note": true,
+                                "type": "General"
+                            },
+                            "routing_rules": []
+                        },
+                        {
+                            "type": "Question",
+                            "id": "confirm-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "options": [
+                                            {
+                                                "label": "Yes",
+                                                "value": "Yes"
+                                            }
+                                        ],
+                                        "id": "confirm-answer",
+                                        "mandatory": false,
+                                        "type": "Checkbox"
+                                    }
+                                ],
+                                "id": "confirm-question",
+                                "title": {
+                                    "text": "Can you confirm that {team} is your favourite team?",
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "team",
+                                            "value": { "source": "answers", "identifier": "favourite-team-answer" }
+                                        }
+                                    ]
+                                },
+                                "interviewer_note": false,
```

Comment:
Not sure we want to go down this route, it would suggest we have to add it to ever question in every schema, and I don't fancy doing that. We do have this pattern of just using true in other places, you could enforce it in validator, but I don't believe we do that anywhere else mind.

Reply:
We did this with `test_radio_detail_answer_visible` schema before, I was told by other devs to check `false` in `radio_visible_answers.spec.js` so I follow the same pattern here.